ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/MainnetBlockProcessor.java

```diff
@@ -158,6 +158,8 @@ private boolean rewardCoinbase(
     if (blockReward.isZero()) {
```

Comment:
would be nice if we could dedupe this method :D, but probably out of scope for this change

Reply:
yes, out of scope I'de say.  Possibly could become static methods.